ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/inmemory/InMemoryLedger.scala

```diff
@@ -100,8 +99,9 @@ class InMemoryLedger(
 
   // mutable state
   private var acs = acs0
-  private var deduplicator = Deduplicator()
   private var ledgerConfiguration: Option[Configuration] = None
+  private var commands: scala.collection.mutable.Map[String, CommandDeduplicationEntry] =
```

Comment:
A `var` pointing to a `mutable.Map`? Seems odd. Can you make this a `val` instead?